content/en/docs/setup/independent/high-availability.md

```diff
@@ -53,296 +53,304 @@ For **Option 2**: you can skip to the next step. Any reference to `etcd0`, `etcd
+                      ]
+                  },
+                  "client": {
+                      "expiry": "43800h",
+                      "usages": [
+                          "signing",
+                          "key encipherment",
+                          "client auth"
+                      ]
+                  },
+                  "peer": {
+                      "expiry": "43800h",
+                      "usages": [
+                          "signing",
+                          "key encipherment",
+                          "server auth",
+                          "client auth"
+                      ]
+                  }
+              }
+          }
+      }
+      EOF
+      ```
+
+      ```none
+      cat >ca-csr.json <<EOF
+      {
+          "CN": "etcd",
+          "key": {
+              "algo": "rsa",
+              "size": 2048
+          }
+      }
+      EOF
+      ```
+
+      {{< note >}}**Optional:** You can modify `ca-csr.json` to add a section for `names`. See [the CFSSL wiki](https://github.com/cloudflare/cfssl/wiki/Creating-a-new-CSR) for an example.{{< /note >}}
+
+3.  Next, generate the CA certs like so:
+
+      ```shell
+      cfssl gencert -initca ca-csr.json | cfssljson -bare ca -
+      ```
 
 ### Generate etcd client certs
 
-1. Generate the client certificates.
+While on `etcd0`, run the following to generate the client certs:
 
-   While on `etcd0`, run the following:
+```none
+cat >client.json <<EOF
+{
+    "CN": "client",
+    "key": {
+        "algo": "ecdsa",
+        "size": 256
+    }
+}
+EOF
+```
 
-   ```shell
-   cat >client.json <<EOF
-   {
-       "CN": "client",
-       "key": {
-           "algo": "ecdsa",
-           "size": 256
-       }
-   }
-   EOF
-   ```
-   ```shell
-   cfssl gencert -ca=ca.pem -ca-key=ca-key.pem -config=ca-config.json -profile=client client.json | cfssljson -bare client
-   ```
+```shell
+cfssl gencert -ca=ca.pem -ca-key=ca-key.pem -config=ca-config.json -profile=client client.json | cfssljson -bare client
+```
 
-This should result in `client.pem` and `client-key.pem` being created.
+This creates `client.pem` and `client-key.pem`.
 
 ### Create SSH access
 
 In order to copy certs between machines, you must enable SSH access for `scp`.
 
-1. First, open new tabs in your shell for `etcd1` and `etcd2`. Ensure you are SSHed into all three machines and then run the following (it will be a lot quicker if you use tmux syncing - to do this in iTerm enter `cmd+shift+i`):
+1.  First, open new tabs in your shell for `etcd1` and `etcd2`. Ensure you are
+    SSHed into all three machines and then run the following (it will be a lot
+    quicker if you use tmux syncing - to do this in iTerm enter `cmd+shift+i`):
 
-   ```shell
-   export PEER_NAME=$(hostname)
-   export PRIVATE_IP=$(ip addr show eth1 | grep -Po 'inet \K[\d.]+')
-   ```
+      ```shell
+      export PEER_NAME=$(hostname)
+      export PRIVATE_IP=$(ip addr show eth1 | grep -Po 'inet \K[\d.]+')
+      ```
```

Comment:
can we just leave everything aligned to the left? i don't think it looks bad without it.
once the ordered list bugs are fixed we just have to modify the digits and the extra space and the alignment will be auto-added; if i understand this correctly, that is...

Reply:
I don't think leaving everything aligned to the left is necessary. The sublist bug only affects two code blocks and we've edited it in such a way that they're no longer affected. So we have the list indentation being rendered correctly and there are no sublist bugs. 

And we have no idea when a fix to this bug will make its way into Hugo.

Reply:
Leaving everything aligned to the left seems to sometimes work and sometimes not. Indenting everything a level seems to always work. I think we have to teach contributors how to contribute docs in the way that creates the least amount of cognitive load, so it's easier to give guidance to always indent things inside lists another level.